### PR TITLE
Create CRDA GH Action.

### PR DESCRIPTION
**Pull request by jparsai**
This PR will create a GH Action for CRDA which can be published to GH Action Marketplace for public use.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,121 @@
...
+## Overview
+The CRDA-analysis action is an interface to Red Hat CodeReady Dependency Analytics platform. 
+It provides vulnerability and compliance analysis for your applications dependencies, along with recommendations to address security vulnerabilities and licensing issues.
+
+This action reflects [Dependency Analytics VS Code plugin](https://marketplace.visualstudio.com/items?itemName=redhat.fabric8-analytics) for Github Actions.
+
+## Prerequisite
+
+#### 1. Manifest File
+This acion scans manifest file (ex. requirements.txt, requirements-dev.txt etc.) for collecting a list of dependencies, hence prior to this action, target manifest file must be placed by user into workspace.
```
Comment (prashbnair) [inline README.md]:
```suggestion
This action scans manifest file (ex. requirements.txt, requirements-dev.txt etc.) for collecting a list of dependencies, hence prior to this action, target manifest file must be placed by user into workspace. 
```
```suggestion
This acion scans manifest file (ex. requirements.txt, requirements-dev.txt etc.) for collecting a list of dependencies, hence prior to this action, target manifest file must be placed by user into workspace. 
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,121 @@
...
+The path to manifest is passed as parameters to action.
+
+#### 2. Dependency Installation
+Users need to install dependencies from manifest into a directory of workspace. 
+Location of installation directory in workspace is required as parameter in the action.
+
+#### 3. Adding Secret
+To authenticate a user this action uses CRDA user key. 
+This key should be saved in secrets, but before that users need to generate it.<br />
+This is just one time activity to perform.
```
Comment (prashbnair) [inline README.md]:
```suggestion
This is a one time activity to perform. 
```
```suggestion
This is just one time activity to perform. 
```

**Review by prashbnair — CHANGES_REQUESTED** [review]

**Comment by prashbnair** [comment]
Looks fine, minor issues in README

**Comment by jparsai** [comment]
Changes are done as requested, Merging the PR.

**Pull request merged**